packages/grafana-data/src/types/config.ts

```diff
@@ -135,9 +135,10 @@ export interface BootData {
   user: CurrentUserDTO;
   settings: GrafanaConfig;
   navTree: NavLinkDTO[];
-  themePaths: {
-    light: string;
+  assets: {
```

Comment:
I like most of this... but would be better if the frontend had less knowlege of the assets

Ideally the checker can be a generic "did things change" without exposing the css/js structures.  Also we will want the same thing to work for frontend plugin changes 🤔